### PR TITLE
- add the loading indicator on the community index page

### PR DESCRIPTION
**Pull request by quochuy**
- allow the community index list to expand full width
- align the top of community index list with the top of side bars

### CODE REVIEW

File: src/app/components/pages/CommunitiesIndex.jsx [hunk 1]
```diff
@@ -19,20 +20,25 @@ export default class CommunitiesIndex extends React.Component {
             communities_idx,
             username,
             walletUrl,
+            loading,
```
Comment (gl2748) [inline src/app/components/pages/CommunitiesIndex.jsx]: Is this being used anywhere?

File: src/app/components/pages/CommunitiesIndex.jsx [hunk 2]
```diff
@@ -19,20 +20,25 @@ export default class CommunitiesIndex extends React.Component {
...
-        if (communities_idx.length == 0) {
+        if (communities_idx.size === 0) {
             return (
                 <center>
-                    <h5>Loading...</h5>
+                    <LoadingIndicator
+                        style={{ marginBottom: '2rem' }}
+                        type="circle"
+                    />
+                    Loading...
```
Comment (gl2748) [inline src/app/components/pages/CommunitiesIndex.jsx]: Probably don't need to show the text 'Loading...' if the spinner is being shown.

File: src/app/components/pages/CommunitiesIndex.jsx [hunk 3]
```diff
@@ -19,20 +20,25 @@ export default class CommunitiesIndex extends React.Component {
...
                 </center>
             );
         }
 
         const role = comm =>
             comm.context &&
-            comm.context.role != 'guest' && (
+            comm.context.role !== 'guest' && (
```
Comment (gl2748) [inline src/app/components/pages/CommunitiesIndex.jsx]: 👍

File: src/app/components/pages/PostsIndex.scss [hunk 4]
```diff
@@ -199,7 +199,7 @@
 
 .articles {
   font-family: "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
-  padding: 0.25em 1em;
+  padding: 0em 1em;
```
Comment (gl2748) [inline src/app/components/pages/PostsIndex.scss]: I would prefer not to include style changes in this PR.  Please submit a seperate PR with screenshots illustrating the change.

File: src/app/components/pages/PostsIndex.scss [hunk 5]
```diff
@@ -212,7 +212,7 @@
     font-family: sans-serif;
   }
   @include MQ(M) {
-    padding: 0.25em 1em;
+    padding: 0em 1em;
```
Comment (gl2748) [inline src/app/components/pages/PostsIndex.scss]: I would prefer not to include style changes in this PR.  Please submit a separate PR with screenshots illustrating the change.

File: src/app/components/pages/PostsIndex.scss (hunk 4 above)
Comment (quochuy) [reply src/app/components/pages/PostsIndex.scss]: I'm going to close this PR and open two new ones:
- https://github.com/steemit/condenser/pull/3647
- https://github.com/steemit/condenser/pull/3648

**Comment by quochuy** [comment]
Split into two PR
#3647
#3648
